### PR TITLE
add slskd

### PR DESCRIPTION
**Pull request by f1refa11**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/slskd.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# software name
+name: "slskd"
+website_url: "https://github.com/slskd/slskd"
+source_code_url: "https://github.com/slskd/slskd"
+description: "A modern client-server application for the Soulseek file sharing network. "
```
Comment (TheGiraffe3) [inline software/slskd.yml:5]:
```suggestion
description: "A modern client-server application for the Soulseek file sharing network."
```

File: software/slskd.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# software name
```
Comment (TheGiraffe3) [inline software/slskd.yml:1]:
```suggestion
```

File: software/slskd.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+name: "slskd"
+website_url: "https://github.com/slskd/slskd"
+source_code_url: "https://github.com/slskd/slskd"
+description: "A modern client-server application for the Soulseek file sharing network."
+licenses:
+  - AGPL-3.0
+platforms:
+  - C#
```
Comment (Rabenherz112) [inline software/slskd.yml:8]: Add Docker accordingly to your installation documentation
```suggestion
platforms:
  - Docker
  - C#
```

File: software/slskd.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+source_code_url: "https://github.com/slskd/slskd"
+description: "A modern client-server application for the Soulseek file sharing network."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Docker
+  - C#
+tags:
+  - File Transfer - Peer-to-peer Filesharing
+depends_3rdparty: true
```
Comment (nodiscc) [inline software/slskd.yml:12]:
```suggestion
```

As far as I know Soulseek is purely peer-to-peer, so there is no dependency on a non-free network service. I don't think this applies.

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: software/slskd.yml (hunk 4 above)
Comment (f1refa11) [reply software/slskd.yml:12]: you can't use slskd and nicotine+ without an account on Soulseek and the connection to it.

Comment (nodiscc) [reply software/slskd.yml:12]: > https://en.wikipedia.org/wiki/Soulseek#Central_server
> Soulseek depends on a pair of central servers [...] these central servers are key to coordinating searches and hosting chat rooms, they do not actually play a part in the transfer of files between users,


I didn't know that, thanks

**Pull request merged**